stock_warehouse_resupply_route_push/tests/test_stock_warehouse_resupply_route_push.py

```diff
@@ -257,6 +257,7 @@ def test_05_recreate_routes_assignation(self):
         wh_4.route_ids += r_wh_3_to_wh4
         # Change to push and recreate routes:
         wh_4.write({"resupply_wh_push": True})
+        self.env.invalidate_all()  # product_route_profile compatibility
```

Comment:
This should not occur here. There is a missing depend for cache invalidation or the invalidation should occur in the module that requires it, but not here.